test/unit/services/sshAdapter_spec.js

```diff
@@ -0,0 +1,285 @@
+"use strict";
+/*jslint plusplus: true */
```

Comment:
I had tried without that initially. I got an error  Unexpected use of '++'. The jshint.json file already has plusplus set to true though